Source/WebCore/workers/service/background-fetch/BackgroundFetchUIOptions.h

```diff
@@ -31,6 +31,13 @@
 namespace WebCore {
 
 struct BackgroundFetchUIOptions {
+    BackgroundFetchUIOptions() = default;
+    BackgroundFetchUIOptions(Vector<ImageResource>&& icons, String&& title)
+        : icons(WTFMove(icons))
+        , title(WTFMove(title))
+    {
+    }
+
```

Comment:
Is there any way to test this kind of thing?

I ask mainly because I have a forthcoming change that will be moving all these IDLDictionary backing structs to be aggregates (which is needed to support non-default initializable IDL dictionaries, such as ones with a Ref<> member).

The alternative for this case would be to just add default values for the members:

```c++
    Vector<ImageResource> icons = { };
    String title = { };
```

(or something like that).

Reply:
The clang static analyzer found these.  I'm not aware of any way to write a test, as that would depend on setting "uninitialized" fields to a specific pattern that otherwise doesn't appear in the code.  (It's possible to do that with ASan or `MallocScribble=1` (which now also controls `MallocPreScribble`) or clang's heap/stack auto-initialization, but that would also require a special build.)

I think using the clang static analyzer is probably the best way to go using the optional [optin.cplusplus.UninitializedObject](https://clang.llvm.org/docs/analyzer/checkers.html#optin-cplusplus-uninitializedobject-c) checker as we will soon be standing up EWS bots and a post-commit bot to check static analyzer results (initially for WebKit checkers, but we could expand to other checkers later).